daily-challenges/2024-11/13/ththao272.spec.ts

```
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+
+const playerName: string = 'ththao27';
+test('the solution 13-11-2024', async ({ page }) => {
+    await page.goto('https://material.playwrightvn.com/games/002-pikachu.html');
+    await page.fill('#playerName', playerName);
+    await page.click('//button[@onclick="startGame()"]');
+    await expect(page.locator('#playerInfo')).toHaveText(`Người chơi: ${playerName}`);
+
+    const texts = await page.locator('//div[@class="outer"]').allInnerTexts();
+    let setOuter: Set<string> = new Set(texts);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'setOuter' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)